eng/helix/content/runtests.cmd

```diff
@@ -21,8 +21,8 @@ echo.
 
 set exit_code=0
 
-echo "Running tests: $HELIX_CORRELATION_PAYLOAD/HelixTestRunner/HelixTestRunner.exe --target %$target% --runtime %$aspRuntimeVersion% --queue %$queue% --arch %$arch% --quarantined %$quarantined% --helixTimeout %$helixTimeout% --playwright $installPlaywright"
-$HELIX_CORRELATION_PAYLOAD/HelixTestRunner/HelixTestRunner.exe --target %$target% --runtime %$aspRuntimeVersion% --queue %$queue% --arch %$arch% --quarantined %$quarantined% --helixTimeout %$helixTimeout% --playwright $installPlaywright
+echo "Running tests: %HELIX_CORRELATION_PAYLOAD%/HelixTestRunner/HelixTestRunner.exe --target %$target% --runtime %$aspRuntimeVersion% --queue %$queue% --arch %$arch% --quarantined %$quarantined% --helixTimeout %$helixTimeout% --playwright $installPlaywright"
+%HELIX_CORRELATION_PAYLOAD%/HelixTestRunner/HelixTestRunner.exe --target %$target% --runtime %$aspRuntimeVersion% --queue %$queue% --arch %$arch% --quarantined %$quarantined% --helixTimeout %$helixTimeout% --playwright $installPlaywright
```

Comment:
Wow❕ I totally missed the use of `$` at the start of this line in our offline conversation. Please also
```suggestion
%HELIX_CORRELATION_PAYLOAD%/HelixTestRunner/HelixTestRunner.exe --target %$target% --runtime %$aspRuntimeVersion% --queue %$queue% --arch %$arch% --quarantined %$quarantined% --helixTimeout %$helixTimeout% --playwright %$installPlaywright%
```